phpbb_poster.py

```diff
@@ -20,6 +20,8 @@
 login_payload['password'] = config[forum]['password']
 
 def grab_secrets(soup):
+    # DEBUG: print(soup)
```

Comment:
Having these commented out for now is OK but there's a way to do this better involving logging, see https://realpython.com/python-logging/, https://www.loggly.com/ultimate-guide/python-logging-basics/,  and https://docs.python.org/3/library/logging.html for future reference.